### PR TITLE
Added test to Bullet and Gun

### PR DESCRIPTION
**Pull request by alvarogerardo007**

### CODE REVIEW

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Gun.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public void paint(Graphics graphic) {
     public void keyPressed(KeyEvent e) {
         if (e.getKeyCode() == KeyEvent.VK_SPACE) {
             if (!bullet.bulletStatus()) {
-                bullet.fire();
+                fire();
                 bullet.setPosX(ship.getPosX());
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Gun.java]: please remove unnecessary spaces

File: src/test/java/org/fundacionjala/at15/spaceinvaders/BulletTest.java [hunk 2]
```diff
@@ -1,18 +1,76 @@
...
+    }
+    @Test
+    public void itShouldMoveTheBullet() {
+        int posX = 10;
+        int posY = 500;
+        boolean fired = true;
+        Bullet bullet = new Bullet(posX, posY, fired);
+        bullet.move();
+        assertEquals(480, bullet.getPosY());
+
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/spaceinvaders/BulletTest.java]: please remove unnecessary spaces

File: src/test/java/org/fundacionjala/at15/spaceinvaders/GunTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import static org.fundacionjala.at15.spaceinvaders.Constants.Player.*;
+public class GunTest {
+
+    @Test
+    public void itShouldShooted() {
+        Ship ship = new Ship(START_X,START_Y);
+        Gun gun = new Gun(ship);
+
+        assertEquals(false,gun.shooted());
+
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/spaceinvaders/GunTest.java]: please remove unnecessary spaces

**Review by adhes14 — DISMISSED** [review]
please read the comments I left you

**Review by adrianovi — APPROVED** [review]

**Pull request merged**